core/kernel/classes/class.Container.php

```diff
@@ -26,21 +26,7 @@
  * @access public
  * @author patrick.plichart@tudor.lu
  * @package generis
-
  */
 class core_kernel_classes_Container extends common_Object
 {
-
-
-    /**
-     * Short description of method __serialize
-     *
-     * @access public
-     * @author firstname and lastname of author, <author@example.org>
-     * @return string
-     */
-    public function __serialize()
```

Comment:
The function is causing a php error as its signature does not match
https://www.php.net/manual/en/language.oop5.magic.php#object.serialize
hence breaking the unit test. If it were used anywhere else it would break that code too.

So while in general I agree that changes to public functions represent backward breaking changes, in this case I actually restored the functionality: https://github.com/oat-sa/generis/pull/799/files#diff-19cb0be98baad0241be1c7fc43f65ea7R770